app/views/ips/_activation_notice.html.erb

```diff
@@ -1,9 +1,6 @@
-<div class="govuk-grid-column-two-thirds">
-  <div class="govuk-inset-text">
-    <h4 class="govuk-heading-s">24 Hours to Activate</h4>
-    <p class="govuk-body">
-      You can configure your access points immedately but it will take 24 hours
-      for your new IP to be activated.
-    </p>
-  </div>
+<div class="govuk-inset-text">
+  <h4 class="govuk-heading-s">New locations are not immediately available</h4>
```

Comment:
We should stick with IP terminology until we actually introduce locations / sites.

Reply:
Sorry, had branched off the other branch before these changes - fixed it now.